flink-batch-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/DerbyUtil.java

```diff
@@ -19,7 +19,6 @@
 
 import java.io.OutputStream;
 
-@SuppressWarnings("unused")
```

Comment:
Now I'll use the class to generate the value of the_ derby.stream.error.field_ property